src/Aspire.Hosting/DistributedApplicationBuilder.cs

```diff
@@ -76,6 +76,7 @@ public DistributedApplicationBuilder(DistributedApplicationOptions options)
 
         _innerBuilder.Logging.AddFilter("Microsoft.Hosting.Lifetime", LogLevel.Warning);
         _innerBuilder.Logging.AddFilter("Microsoft.AspNetCore.Server.Kestrel", LogLevel.Error);
+        _innerBuilder.Logging.AddFilter("Aspire.Hosting.Dashboard", LogLevel.Error);
```

Comment:
Not sure we want to do this. Maybe we should update the templates? The downside is that upgrading won't have it and there will be LOTS of noise.